Makefile

```
@@ -13,6 +13,7 @@ flatten              :; hevm flatten --source-file "src/DssSpell.sol" > out/flat
 diff-deployed-spell  :; ./scripts/diff-deployed-dssspell.sh $(spell)
 check-deployed-spell :; ./scripts/check-deployed-dssspell.sh
 cast-spell           :; ./scripts/cast-dssspell.sh $(spell)
+cast-on-tenderly     :; cd ./scripts/cast-on-tenderly/ && npm i && node index.js $(spell)
```

Maybe switch back to the original directory after executing:

```suggestion
cast-on-tenderly     :; cd ./scripts/cast-on-tenderly/ && npm i && node index.js $(spell); cd -
```

---

I've updated it, thanks! FYI: in order to pass arguments to the underlying npm script, one have to use `--`, [see docs](https://docs.npmjs.com/cli/v8/commands/npm-run-script)